Disable opal_path_nfs test in OpenMPI 5.0.3 as this test can easily fail on some systems,
when NFS mounts are used. Generally, this test is flaky, which may prevent users
from installing OpenMPI for no apparent reason.

diff --git a/test/util/Makefile.am b/test/util/Makefile.am
index e5ad472..33d63c4 100644
--- a/test/util/Makefile.am
+++ b/test/util/Makefile.am
@@ -38,7 +38,6 @@ AM_CPPFLAGS = -I$(top_srcdir)/test/support
 
 check_PROGRAMS = \
 	opal_bit_ops \
-	opal_path_nfs \
 	bipartite_graph \
         opal_sha256
 
@@ -80,11 +79,11 @@ opal_bit_ops_LDADD = \
         $(top_builddir)/test/support/libsupport.a
 opal_bit_ops_DEPENDENCIES = $(opal_path_nfs_LDADD)
 
-opal_path_nfs_SOURCES = opal_path_nfs.c
-opal_path_nfs_LDADD = \
-        $(top_builddir)/opal/lib@OPAL_LIB_NAME@.la \
-        $(top_builddir)/test/support/libsupport.a
-opal_path_nfs_DEPENDENCIES = $(opal_path_nfs_LDADD)
+# opal_path_nfs_SOURCES = opal_path_nfs.c
+# opal_path_nfs_LDADD = \
+#         $(top_builddir)/opal/lib@OPAL_LIB_NAME@.la \
+#         $(top_builddir)/test/support/libsupport.a
+# opal_path_nfs_DEPENDENCIES = $(opal_path_nfs_LDADD)
 
 #opal_os_path_SOURCES = opal_os_path.c
 #opal_os_path_LDADD = \
